### DISCUSSION TITLE
Request for research published as PDF papers on Research Gate

### DISCUSSION THREAD

**Discussion by balupton** [Tribler/tribler#5919] [Ideas]
Currently there are about 50 research paper links on [the wiki](https://github.com/Tribler/tribler/wiki), and about a hundred more throughout the issues.

[Research Gate](https://www.researchgate.net) is a community of academics sharing their research, it would be great if the tribler authors could share their research there, so it can be followed, easily downloaded, easily cited, etc.

I'm planning to work my way through all of the papers and discuss them on my [study group's youtube channel and podcast](https://bevry.me/youtube), so that I can contribute to the space rather than just leverage it, and also expose the space to other's.

So would be good to be able to formerly link people to that research community and get a paper that I can annotate and discuss with others.

Another benefit is that about 1/4 of the papers linked through the wiki and the issues, result in 404 not founds, so uploading the papers to research gate and using those links will not have this problem.

---

Here are the papers that I've been able to find on research gate:

https://ieeexplore.ieee.org/abstract/document/5160954
=>
https://www.researchgate.net/publication/228871839_BarterCast_A_practical_approach_to_prevent_lazy_freeriding_in_P2P_networks

https://dash.harvard.edu/handle/1/4907301
=>
https://www.researchgate.net/publication/228843446_On_the_Sybil-Proofness_of_Accounting_Mechanisms

https://dl.acm.org/doi/10.5555/1838206.1838239
=>
https://www.researchgate.net/publication/221454185_Hybrid_transitive_trust_mechanisms

https://dl.acm.org/doi/10.14778/1929861.1929864
=> full text requested
https://www.researchgate.net/publication/262282224_Fast_incremental_and_personalized_PageRank

https://www.aaai.org/ocs/index.php/AAAI/AAAI10/paper/viewPaper/1802
=>
https://www.researchgate.net/publication/221606696_Accounting_Mechanisms_for_Distributed_Work_Systems

https://science.sciencemag.org/content/314/5805/1560
=> full text requested
https://www.researchgate.net/publication/334062374_Five_Rules_for_the_Evolution_of_Cooperation

https://www.nature.com/articles/nature03204
=>
https://www.researchgate.net/publication/8069911_Evolutionary_Dynamics_on_Graphs

http://pages.stern.nyu.edu/~lcabral/reputation/Reputation_June05.pdf
=> requested upload
https://www.researchgate.net/publication/228389641_The_economics_of_trust_and_reputation_a_primer

https://www.nature.com/articles/nature04131
=>
https://www.researchgate.net/publication/7514734_Evolution_of_Indirect_Reciprocity

https://ieeexplore.ieee.org/document/994181
=>
https://www.researchgate.net/publication/224075946_A_computational_model_of_trust_and_reputation

https://papers.ssrn.com/sol3/papers.cfm?abstract_id=2519367
=>
https://www.researchgate.net/publication/314406810_Can_We_Stabilize_the_Price_of_a_Cryptocurrency_Understanding_the_Design_of_Bitcoin_and_Its_Potential_to_Compete_with_Central_Bank_Money

https://ieeexplore.ieee.org/document/8696786
=>
https://www.researchgate.net/publication/332677616_Real-time_Money_Routing_by_Trusting_Strangers_with_your_Funds

https://www.usenix.org/system/files/conference/usenixsecurity15/sec15-paper-soska-updated.pdf
=>
https://www.researchgate.net/publication/308057059_Measuring_the_longitudinal_evolution_of_the_online_anonymous_marketplace_ecosystem

https://cryptome.org/2014/04/tor-vulns-attacks.pdf
=>
https://www.researchgate.net/publication/261725828_The_fifteen_year_struggle_of_decentralizing_privacy-enhancing_technology

https://crypto.stanford.edu/%7Edabo/papers/2dnf.pdf
=>
https://www.researchgate.net/publication/221354138_Evaluating_2-DNF_Formulas_on_Ciphertexts

https://dash.harvard.edu/bitstream/handle/1/11856150/Seuken_AccountingMechanisms.pdf
=>
https://www.researchgate.net/publication/221606696_Accounting_Mechanisms_for_Distributed_Work_Systems

https://www.ifi.uzh.ch/ce/publications/Accounting_Mechanisms.pdf
=>
https://www.researchgate.net/publication/229059473_Work_Accounting_Mechanisms_Theory_and_Practice

https://people.csail.mit.edu/nickolai/papers/gilad-algorand-eprint.pdf
=>
https://www.researchgate.net/publication/320362651_Algorand_Scaling_Byzantine_Agreements_for_Cryptocurrencies

https://dl.acm.org/doi/10.1145/2695664.2695868
=>
https://www.researchgate.net/publication/273379999_Trust-based_Collection_of_Information_in_Distributed_Reputation_Networks

https://dl.acm.org/doi/10.1145/1810617.1810683
=>
https://www.researchgate.net/publication/221267161_On_the_robustness_of_Google_Scholar_against_spam

https://dl.acm.org/doi/10.1145/1189702.1189703
=>
https://www.researchgate.net/publication/200110713_A_reference_collection_for_web_spam

https://arxiv.org/abs/1203.4011
=>
https://www.researchgate.net/publication/221405022_Understanding_Sampling_Style_Adversarial_Search_Methods
https://www.researchgate.net/publication/221710951_Understanding_Sampling_Style_Adversarial_Search_Methods

http://publicatio.bibl.u-szeged.hu/2509/1/SRE-p2p11-final-version.pdf
=>
https://www.researchgate.net/publication/224261449_Fast_download_but_eternal_seeding_The_reward_and_punishment_of_Sharing_Ratio_Enforcement

https://www.ifi.uzh.ch/ce/publications/Accounting_Mechanisms.pdf
=>
https://www.researchgate.net/publication/229059473_Work_Accounting_Mechanisms_Theory_and_Practice

http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.76.3107&rep=rep1&type=pdf
=>
https://www.researchgate.net/publication/221160480_The_Bittorrent_P2P_File-Sharing_System_Measurements_and_Analysis

---

haven't yet gone through these, as there are so many:
https://github.com/Tribler/tribler/issues/5221#issuecomment-616185473

---

So there is a Tribler project on research gate:
https://www.researchgate.net/project/Tribler

However, it doesn't have many of the papers listed on it, only one!

**Comment by devos50** [comment]
Thanks for your suggestion! Indeed, making our papers public and organized is an important job for visibility and impact.

We already prefer "open-access" journals and conferences. After publication of a paper, we have to upload it to [PURE](https://research.tudelft.nl), which is a TU Delft system for publication management. I try to upload these papers as soon as possible after acceptance. I can probably link to these uploaded papers in Researchgate, I'll have a look at that 👍 

Unfortunately, we also have quite a few papers that cannot benefit from open access. For example, our initial TrustChain paper, which has been published in Elsevier FGCS, cannot be uploaded publicly because of contractual issues.

**Reply by balupton** [reply]
Cool, if you need an invite, let me know.
